Add overnight battery loan form

RealOrangeOne opened the pull request:
Adds a form for overnight battery loans for https://github.com/srobo/tasks/issues/213.

Also required for https://github.com/srobo/runbook/pull/20

RealOrangeOne committed:
Add overnight battery loan form

RealOrangeOne committed:
Build doc on CI

RealOrangeOne committed:
Add column for battery bag return

PeterJCLaw commented on overnight-battery-loan/form.tex:
```diff
@@ -0,0 +1,48 @@
+\begin{center}
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
```
Not sure if it's intentional -- this table is missing its top line.
```suggestion
\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
\hline
```

PeterJCLaw commented on overnight-battery-loan/Makefile:
```diff
@@ -0,0 +1,6 @@
+all: form.pdf
+
+
+form.pdf: form.tex
```
We're potentially going to end up with a lot of `form.pdf` files here, perhaps we could name them the same as their folder instead?

PeterJCLaw commented on overnight-battery-loan/form.tex:
```diff
@@ -0,0 +1,48 @@
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Charger Part Number} & \textbf{Battery Bag Part Number} & \textbf{Battery 1 Part Number} & \textbf {Battery 2 Part Number} & \textbf{Charger Returned?} & \textbf{Battery Bag Returned?} & \textbf{Battery 1 Returned?} & \textbf{Battery 2 Returned?} \\
```
Does LaTeX let us wrap these so that we don't have quite such a long line?
```suggestion
    \textbf{TLA} &
    \textbf{Charger Part Number} &
    \textbf{Battery Bag Part Number} &
    \textbf{Battery 1 Part Number} &
    \textbf{Battery 2 Part Number} &
    \textbf{Charger Returned?} &
    \textbf{Battery Bag Returned?} &
    \textbf{Battery 1 Returned?} &
    \textbf{Battery 2 Returned?} \\
```

PeterJCLaw commented on overnight-battery-loan/form.tex:
```diff
@@ -0,0 +1,48 @@
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Charger Part Number} & \textbf{Battery Bag Part Number} & \textbf{Battery 1 Part Number} & \textbf {Battery 2 Part Number} & \textbf{Charger Returned?} & \textbf{Battery Bag Returned?} & \textbf{Battery 1 Returned?} & \textbf{Battery 2 Returned?} \\
```
Should we also log the part number of the charger power supply?

RealOrangeOne replied:
i'd forgotten these are inventorised separately, we probably should then!

PeterJCLaw committed:
Reflow this

PeterJCLaw committed:
Rework for portrait layout

This inlines the cells for whether an item was returned into being
just below the part number, which avoids the table being particularly
wide while also (hopefully) making it easier to see at a glance what
hasn't been returned.

PeterJCLaw commented on overnight-battery-loan/form.tex:
```diff
@@ -0,0 +1,48 @@
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|Y|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Charger Part Number} & \textbf{Battery Bag Part Number} & \textbf{Battery 1 Part Number} & \textbf {Battery 2 Part Number} & \textbf{Charger Returned?} & \textbf{Battery Bag Returned?} & \textbf{Battery 1 Returned?} & \textbf{Battery 2 Returned?} \\
```
This table is pretty wide, how about re-working the returned boxes to being inline?

PeterJCLaw replied:
I've put a version of this in https://github.com/srobo/kit-coordination-docs/pull/7

PeterJCLaw reviewed (commented):
Mostly good, see inline comments

RealOrangeOne committed:
Merge pull request #7 from PeterJCLaw/overnight-battery-loan-portrait

Portrait version of overnight battery loan form

RealOrangeOne committed:
Merge branch 'master' into overnight-battery-loan

RealOrangeOne committed:
Give form files a relevant name

Adimote commented on overnight-battery-loan/overnight-battery-loan.tex:
```diff
@@ -0,0 +1,62 @@
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|Y|Y|}
+    \textbf{TLA} &
+    \textbf{Charger Part Number} &
```
We need a new column for the charger power supply

Adimote reviewed (dismissed):
We're missing a column for the charger power supply

RealOrangeOne committed:
Add column for charger PSU

RealOrangeOne committed:
Merge remote-tracking branch 'origin/master' into overnight-battery-loan

# Conflicts:
#	.circleci/config.yml

PeterJCLaw commented on overnight-battery-loan/overnight-battery-loan.tex:
```diff
@@ -0,0 +1,63 @@
+\begin{table}[h]
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|c|Y|Y|Y|Y|Y|}
+    \textbf{TLA} &
```
Still missing a top line, which I think is fixable via:
```suggestion
\hline
    \textbf{TLA} &
```
